x-pack/plugins/infra/server/lib/alerting/log_threshold/log_threshold_executor.ts

```diff
@@ -163,6 +163,42 @@ export const createLogThresholdExecutor = (libs: InfraBackendLibs) =>
           startedAt.valueOf()
         );
       }
+
+      const { getRecoveredAlerts } = services.alertFactory.done();
+      const recoveredAlerts = getRecoveredAlerts();
+      for (const alert of recoveredAlerts) {
```

Comment:
Thanks for these changes 🙏

Just one small nit: could we move the processing of these recovered alerts to a separate function?

Something like:

```ts
const { getRecoveredAlerts } = services.alertFactory.done();
const recoveredAlerts = getRecoveredAlerts();
processRecoveredAlerts(recoveredAlerts, services);
```